components/PlantItem.js

```diff
@@ -3,18 +3,20 @@ import { View, Text, Image, TouchableHighlight} from 'react-native';
 
 import styles from '../styles/styles';
 
-export default function PlantItem ({ title, image, searchNavigation }) {
-  return(
+export default function PlantItem ({ title, image, searchNavigation, tabNavigation: {  dangerouslyGetParent} }) {
+  return (
     <TouchableHighlight
-      onPress={() => searchNavigation.navigate('PlantInfoPage')}
-    >
+      onPress={() => {
+        searchNavigation.navigate('PlantInfoPage');
+        dangerouslyGetParent().setOptions({ tabBarVisible: false });
```

Comment:
For Reference: Maybe there is a way that we can set this in the tabNavigator instead of toggling back and forth?